sklearn/svm/classes.py

```diff
@@ -594,7 +594,7 @@ def __init__(self, nu=0.5, kernel='rbf', degree=3, gamma=0.0,
 
 
 class SVR(BaseLibSVM, RegressorMixin):
-    """epsilon-Support Vector Regression.
+    """Epsilon-Support Vector Regression.
```

Comment:
Actually this is typically spelled with a lowercase epsilon (ε). I guess that's what the docstring is trying to mimic in ASCII. (Not a big deal, though.)